src/main/java/com/google/devtools/build/lib/remote/Retrier.java

```diff
@@ -100,7 +100,7 @@ enum State {
     State state();
 
     /** Called after an execution failed. */
-    void recordFailure();
+    void recordFailure(Throwable t);
```

Comment:
This is a breaking change so you need to find another way to record ignored errors.

Reply:
@werkt Will update it to Exception.

Reply:
@shirchen for backward compatibility I can define an overloaded method.
```
@deprecated
default void recordFailure() {
   // As there was not default implementation earlier, all implementing classes must have implemented this. 
   throw IllegalState("Not implemented Exception")
}

default void recordFailure(Exception e) {
   // for backward compatibility
   recordFailure() 
}
```